src/constants/typedb.constant.ts

```diff
@@ -0,0 +1,16 @@
+export class DATATYPE_MYSQL {
+  static readonly STRING = 'varchar'
+  static readonly NUMBER = 'int'
+  static readonly DATE = 'date'
+  static readonly TIMESTAMP = 'timestamp'
+  static readonly BOOLEAN = 'boolean'
+  static readonly TEXT = 'text'
+  static readonly JSON = 'json'
+  static readonly ENUM = 'enum'
+}
+export class ENTITY_PROPERTY_LENGTH {
+  static readonly SHORT = 100
+  static readonly MEDIUM = 200
+  static readonly LONG = 500
+  static readonly XLONG = 1000
```

Comment:
```suggestion
  static readonly NUM_100 = 100
  static readonly NUM_200 = 200
  ...
```
Đặt tên như này sẽ dễ hiểu hơn e nha. Khi đọc code cũng sẽ biết đáp án liền